easypost/shipment.py

```diff
@@ -19,11 +20,27 @@
 
 
 class Shipment(AllResource, CreateResource):
-    def regenerate_rates(self) -> "Shipment":
+    @classmethod
+    def create(cls, api_key: Optional[str] = None, carbon_offset: bool = False, **params) -> Any:
+        """Create an Shipment object."""
+        requestor = Requestor(local_api_key=api_key)
+        url = cls.class_url()
+        wrapped_params = {cls.snakecase_name(): params}
+        if carbon_offset:
```

Comment:
Let's remove the `if` statement and always pass in carbon_offset. If the user did not request it, it will be set to False by default.

Reply:
Yes, we could. This will impact about 25 unit tests and we will need to re-record the cassettes

Reply:
Let's do it. I think it's what we decided for the other libs too.